frontend/src/app/libs/landing/store/store.module.ts

```diff
@@ -0,0 +1,9 @@
+import { NgModule } from '@angular/core';
+import { StoreModule } from '@ngrx/store';
+import { EffectsModule } from '@ngrx/effects';
+import { eventsFeatureKey, eventsReducer, EventsEffects } from './events';
+
+@NgModule({
+	imports: [StoreModule.forFeature(eventsFeatureKey, eventsReducer), EffectsModule.forFeature([EventsEffects])],
```

Comment:
`landing` store  is feature store, events just contains in landing store, we need to use `landingFeatureKey`